deepinv/utils/demo.py

```diff
@@ -105,3 +108,31 @@ def load_degradation(name, data_dir, kernel_index=0, download=True):
     kernels = np.load(kernel_path, allow_pickle=True)
     kernel_torch = torch.from_numpy(kernels[kernel_index])  # .unsqueeze(0).unsqueeze(0)
     return kernel_torch
+
+
+def load_image(
+    path=None,
+    url=None,
+    img_size=None,
+    grayscale=False,
+    resize_mode="crop",
+    device="cpu",
+):
```

Comment:
we can add some docstring explaining the function.
I would also rename it to load_url_image